staticd/static_vrf.c

```diff
@@ -76,6 +77,8 @@ static int static_vrf_new(struct vrf *vrf)
 
 static int static_vrf_enable(struct vrf *vrf)
 {
+	static_zebra_vrf_register(vrf);
```

Comment:
Shouldn't we need to unregister the VRF in the `static_vrf_disable()` callback?

Reply:
I did it, but I wonder if the unregister mechanism is not when an instance is leaving ( for instance bgp). because when a vrf is becoming disabled, that means no more events will be exchanged with zebra. So I doubt about the unregistration event, though I added it.